lib/ansible/modules/cloud/azure/azure_rm_aks.py

```diff
@@ -89,6 +89,12 @@
                 description:
                     - The secret password associated with the service principal.
                 required: true
+    enable_rbac:
```

Comment:
if enable_rbac, then user should able to grant permission to  service  principal, so what's next step?

Reply:
I find [this article](https://docs.microsoft.com/en-us/azure/aks/aad-integration) mentions the "Existing non-RBAC enabled AKS clusters cannot currently be updated for RBAC use", we may need to add this information or this link to the documentation

Reply:
ok, updated the doc